app/experimenter/experiments/tests/test_views.py

```diff
@@ -782,3 +782,11 @@ def test_page_loads(self):
             **{settings.OPENIDC_EMAIL_HEADER: user_email},
         )
         self.assertEqual(response.status_code, 200)
+
+
+class Test404View(TestCase):
+    def test_404(self):
```

Comment:
I actually also wanted to create a test for `test_404_dev`:

```
@override_settings(DEBUG=True)
def test_404_dev(self):

  response = self.client.get("/404")
  self.assertTemplateUsed(response, 'nimbus/404.html')
  self.assertEqual(response.status_code, 200)
```
(could also supply a `name` like `name="page-not-found"` in `urls.py` and check for `reverse("page-not-found")`)

Which should work since in `urls.py` we render it `as_view` and the Network tab definitely shows a 200 response, but I get not only a "template not used" assertion error, but a 404 response code. Maybe `@override_settings(DEBUG=True)` doesn't work like how I thought it would.